infrastructure/terraform/terraform.tfvars

```diff
@@ -0,0 +1,12 @@
+project_id  = "konomi-ai"
+
+# Pick a region with low spot VM prices. us-west4 is currently the cheapest.
+# https://cloud.google.com/compute/vm-instance-pricing
+region           = "us-west4"
+zone             = "us-west4-a"
```

Comment:
I would place our service on the east coast instead

Reply:
Easy fix to make, I set the region as `us-west4` because if we want to use spot VMs (for staging), they are priced significantly cheaper.